WolvenKit/Views/Tools/RedTreeView.xaml.cs

```diff
@@ -385,7 +381,14 @@ private void OnDoubleClick(object sender, MouseButtonEventArgs e)
                 return;
             }
 
+            // If the current chunk is not expanded, collapse all of its siblings
+            if (!chunk.IsExpanded && chunk.Parent is ChunkViewModel parent)
```

Comment:
I kinda don't understand this... so if the selected item is not expanded, then we select the parent?